sonic-net/sonic-buildimage
# [202012] [TACACS+] When load-minigraph enable per-command accounting by default

When load-minigraph enable per-command accounting by default.

##### Work item tracking
- Microsoft ADO **(number only)**: 24433713
- 
#### Why I did it
Enable per-command accounting by default.

#### How I did it
Add hard code config in sonic-cfggen.

#### How to verify it
Also pass all current UT.
Manually check the TACACS per-command accounting enabled after load minigraph.

#### Which release branch to backport (provide reason below if selected)
N/A

#### Description for the changelog
When load-minigraph enable per-command accounting by default.

#### A picture of a cute animal (not mandatory but encouraged)
